package.json

```diff
@@ -5,7 +5,8 @@
   "main": "src/main.js",
   "scripts": {
     "start": "vue-cli-service serve",
-    "build": "vue-cli-service build"
+    "build": "vue-cli-service build",
+    "start:s3_bucket": "npm start -- --open --CATALOG_URL=http://localhost:5000/collection.json --STAC_PROXY_URL='s3://pxapi-media-dev/pixelsdata/34146aab-78b5-4b54-a7d1-9b228d2a9fec/data|http://localhost:5000'"
```

Comment:
This shouldn't be here. Remember that the goal of this work is to open a PR into the original project